ios/PurchasesHybridCommon/PurchasesHybridCommonIntegrationTests/BaseIntegrationTests.swift

```diff
@@ -68,13 +67,13 @@ private extension BaseIntegrationTests {
     }
 
     func configurePurchases() {
-        Purchases.configure(withAPIKey: Constants.apiKey,
+        _ = Purchases.configure(apiKey: Constants.apiKey,
                             appUserID: nil,
                             observerMode: false,
-                            userDefaults: self.userDefaults,
-                            storeKit2Setting: Self.storeKit2Setting,
-                            storeKitTimeout: Configuration.storeKitRequestTimeoutDefault,
-                            networkTimeout: Configuration.networkTimeoutDefault,
+                            userDefaultsSuiteName: Constants.userDefaultsSuiteName,
+                            platformFlavor: nil,
```

Comment:
Feels weird to pass in `nil` for flavors, but I see the default is `nil`.

Reply:
Just thinking- do we need to add it to the configuration builder so each hybrid can add it?

Reply:
Ok confirmed, I'm adding.